sdk/storage/inc/datalake/path_client.hpp

```diff
@@ -207,18 +226,70 @@ namespace Azure { namespace Storage { namespace DataLake {
     /**
      * @brief Sets the properties of a resource the path points to.
      * @param options Optional parameters to set the properties to the resource the path points to.
-     * @return PathUpdateResponse
+     * @return SetPathPropertiesResponse
      */
-    PathUpdateResponse SetProperties(
+    SetPathPropertiesResponse SetProperties(
```

Comment:
I remember we've changed this to `SetHttpHeaders`.

Reply:
This API can also set metadata.

Reply:
Please keep aligned with other SDK

Reply:
Sure, changed to SetHttpHeaders and removed the functionality to set metadata.